asserts/pool.go

```diff
@@ -273,6 +362,52 @@ func (p *Pool) phase(ph poolPhase) error {
 	return nil
 }
 
+// AddSequenceToUpdate adds the assertion referenced by toUpdate and all its
+// prerequisites to the Pool as unresolved and as required by the
```

Comment:
this is not true atm but we want to do that either here or in a follow up to avoid round trips.